GitUI/NBugReports/BugReportInvoker.cs

```diff
@@ -188,6 +210,42 @@ void AddIgnoreOrCloseButton()
             }
         }
 
+        private static UserAction ReportFailToLoadAnAssembly(FileNotFoundException exception)
+        {
+            string error = exception.Message;
+            TaskDialogPage pageFailToLoadAssembly = new()
+            {
+                Icon = TaskDialogIcon.Error,
+                Caption = TranslatedStrings.FailToLoadAnAssembly,
+                Heading = TranslatedStrings.FailToLoadAnAssemblyExplanation,
+
+                AllowCancel = false,
+                SizeToContent = true,
+            };
+
+            TaskDialogCommandLinkButton relaunchButton = new(TranslatedStrings.RelaunchGitExtensions);
+            pageFailToLoadAssembly.Buttons.Add(relaunchButton);
+
+            TaskDialogCommandLinkButton reportButton = new(TranslatedStrings.ReportTheIssue);
+            pageFailToLoadAssembly.Buttons.Add(reportButton);
+
+            pageFailToLoadAssembly.Expander = new TaskDialogExpander
+            {
+                CollapsedButtonText = TranslatedStrings.SeeErrorMessage,
+                ExpandedButtonText = TranslatedStrings.HideErrorMessage,
+                Position = TaskDialogExpanderPosition.AfterFootnote,
+                Text = error,
+            };
+
+            TaskDialogButton button = TaskDialog.ShowDialog(OwnerFormHandle, pageFailToLoadAssembly);
+            if (button == relaunchButton)
+            {
+                return UserAction.ReLaunch;
+            }
+
+            return UserAction.ReportIssue;
```

Comment:
can be shortened now:
```suggestion
            return button == relaunchButton ? UserAction.ReLaunch : UserAction.ReportIssue;
```